packages/api/src/schema/api-schema.yaml

```
@@ -244,26 +303,33 @@ components:
         duration:
           type: number
           description: The time taken (in seconds) to make the webhook request
+          example: 0.5
         success:
           type: boolean
           description: Whether the webhook request was successful
+          example: true
         request:
           type: object
           properties:
             url:
               type: string
               description: URL used for the request
+              example: "https://my-service.com/webhook"
             method:
               type: string
               description: HTTP request method
+              example: POST
             headers:
               type: object
               description: HTTP request headers
               additionalProperties:
                 type: string
+              example:
+                Authorization: Bearer 123
```

We don't send auth headers today so this may be confusing. Perhaps this?
```suggestion
                User-Agent: livepeer.studio
```